pages/browseByCategory.jsx

```diff
@@ -0,0 +1,9 @@
+const browsByCategory = () => {
```

Comment:
Since a "page" in next is just a component you should use an uppercase character as first character in your component name.